mods/ra/mod.yaml

```diff
@@ -223,7 +223,7 @@ SoundFormats: Aud, Wav
 
 SpriteFormats: ShpD2, ShpTD, TmpRA, TmpTD, ShpTS
 
-VideoFormats: Vqa
+VideoFormats: Vqa, Wsa
```

Comment:
It seems there are only 3 WSA files in the currently installed assets. You add the TD `general.mix` to the mod manifest (installed beforehand, which is odd). Can you add the relevant MIX file for RA as well?

Reply:
Finally found them.